symtabAPI/src/Object-nt.C

```diff
@@ -2336,7 +2336,7 @@ void Object::insertPrereqLibrary(std::string lib)
            getRegionPermissions() == RP_RWX);
 }
 
-Dyninst::Architecture Object::getArch()
```

Comment:
There should be a way to tell whether this is properly x86 or x86_64 from the file headers, no? May as well get that bit of infrastructure in.